pkg/kubelet/kubelet_volumes.go

```diff
@@ -111,7 +111,7 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(
 			continue
 		}
```

Comment:
I think the we should remove the check len(volumePaths) > 0. If we have this check, any directories exist under /volumes will skip the clean up and  RemoveAllOneFilesystem will not be called. The purpose of checking len(volumePaths) > 0 was to prevent any data loss in case the directories under /volumes are not cleaned up properly. Currently, it is possible that the directories under /volumes are unmounted, but fail to delete. If this happens, cleanupOrphanedPod will fail to delete the pod directory.